test_enum_fix.py

```
@@ -0,0 +1,142 @@
+            }
+        },
+        "required": []
+    }
+
+    # Create schema twice (simulating what happens in the bug)
+    print("\n1. Creating first schema...")
+    Schema1 = model_from_mcp_schema("search_datasets", mcp_schema)
+    field1_annotation = Schema1.model_fields["sortBy"].annotation
+
+    # Extract the actual enum class (might be in a Union)
+    import typing
```

Ruff (0.14.14)

[error] Redefinition of unused `typing` from line 6: `typing` redefined here

Remove definition: `typing`

(F811)